### DISCUSSION TITLE
Need to understand in our secure url , why it is coming http on LOGIN as this user in the different browser

### DISCUSSION THREAD

**Discussion by vaishaliPimcore** [pimcore/pimcore#10941] [Q&A / Support]
### Expected behavior

It is should come in https://website

### Actual behavior

is it coming in http://website

### Steps to reproduce

[](url
![our-project](https://user-images.githubusercontent.com/59276447/144380958-f4c287b6-c953-4486-8131-94de6b2bfaba.png)
![demoPimcore](https://user-images.githubusercontent.com/59276447/144380967-4a40f36b-f59c-4e5e-aab3-3cf5c402dac9.png)
)

Please let me know where I can change for to make URL HTTPS in pimcore10 as same as in demo website.

**Comment by fashxp** [comment]
probably you need to configure trusted proxies? 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Configuration.html#page_Adding-logic-to-the-startup-process
